nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py

```diff
@@ -78,6 +87,7 @@
     'RobertaTokenizer': "vocab.json",
     'BertTokenizer': "vocab.txt",
     'DistilBertTokenizer': "vocab.txt",
+    'CamembertTokenizer': "sentencepiece.bpe.model",
```

Comment:
How about using the same name as in AlbertTokenizer, as they both use SentencePiece model, i.e. `"spiece.model"`?

Reply:
I used the naming convention as present in Camembert model files, see for example https://huggingface.co/camembert-base/tree/main, https://huggingface.co/EMBEDDIA/sloberta/tree/main, https://huggingface.co/cmarkea/distilcamembert-base/tree/main. This seems to differ from Albert models, which use `spiece.model`, see https://huggingface.co/albert-base-v2/tree/main. 

I suggest leaving as is.